ChangeLog.md

```diff
@@ -99,6 +99,8 @@
 
   * Add Python library regex to plbase (Nathan Bowman).
 
+  * Add `all-of-the-above` and `none-of-the-above` attributes to `<pl-multiple-choice>` (Bojin Yao)
```

Comment:
In the future you don't need to update the changelog, we've stopped using it in favor of Git commits when new versions are deployed

Reply:
Oh I see, i was wondering about that haha. I think Contributions page might need an update haha. Thank you!